src/Data/Uniques/sword.lua

```diff
@@ -449,6 +449,7 @@ Variant: Pre 3.0.0
 Variant: Pre 3.16.0
 Variant: Current
 League: Breach
+Requires Level 69
```

Comment:
```suggestion
LevelReq: 69
```
The rest of the file seems to be using this format.